nextcord/types/interactions.py

```diff
@@ -232,7 +232,7 @@ class MessageInteraction(TypedDict):
     id: Snowflake
     type: InteractionType
     name: str
-    user: User
+    user: Union[Member, User]
```

Comment:
Types is supposed to represent Discord's data objects, so this should be just User
```suggestion
    user: User
```
Member can be added in an optional class for consistency, eg.
```py
class _MessageInteractionOptional(TypedDict, total=False):
	member: Member
```
And have MessageInteraction subclass it

Reply:
`member` still needs to be added to the TypedDict (see above)

Reply:
I was doing that, couldnt find my glasses lol💀
I saved changes